db/database.go

```diff
@@ -1734,9 +1734,21 @@ func (db *DatabaseCollectionWithUser) getResyncedDocument(ctx context.Context, d
 }
 
 func (db *DatabaseCollectionWithUser) resyncDocument(ctx context.Context, docid, key string, regenerateSequences bool, unusedSequences []uint64) (updatedHighSeq uint64, updatedUnusedSequences []uint64, err error) {
+	startTime := time.Now()
 	var updatedDoc *Document
 	var shouldUpdate bool
 	var updatedExpiry *uint32
+	defer func() {
+		functionTime := time.Since(startTime).Milliseconds()
+		var bytes int
+		if updatedDoc != nil {
+			bytes = len(updatedDoc._rawBody)
+		} else {
+			return
+		}
+		stat := calculateImportCompute(int64(bytes), functionTime)
+		db.dbStats().DatabaseStats.ImportProcessCompute.Add(stat)
+	}()
```

Comment:
Question for the group is whether we should include `UpdateAllPrincipals` in this computation as well, which is done for the whole group. We could bundle up the whole resync process as one computation rather than going doc by doc.